pkg/pod/network.go

```diff
@@ -80,3 +83,37 @@ func StaticIPBondAnnotationWithInterface(
 
 	return append(annotation, bond[0])
 }
+
+// StaticIPMultiNetAnnotation defines static ip address network annotation for pod object.
+func StaticIPMultiNetAnnotation(name string, ipAddr []string) *multus.NetworkSelectionElement {
+	return &multus.NetworkSelectionElement{
+		Name:      name,
+		IPRequest: ipAddr,
+	}
+}
+
+// StaticIPMultiNetDualStackAnnotation defines static ip address network annotation for pod object.
+func StaticIPMultiNetDualStackAnnotation(sriovNets, ipAddr []string) []*multus.NetworkSelectionElement {
+	annotation := []*multus.NetworkSelectionElement{}
+
+	// Verify ipAddr has an even number of IP addresses.
+	if len(ipAddr)%2 != 0 {
```

Comment:
Probably we should add more validations: sriovNets, ipAddr []string are not empty

Reply:
checking that they are not empty